connector-packager/tests/test_xsd_validator.py

```diff
@@ -65,3 +65,21 @@ def test_validate_single_file(self):
         logging.debug("test_validate_single_file xml violations:")
         for violation in xml_violations_buffer:
             logging.debug(violation)
+
+    def test_validate_vendor_prefix(self):
+
+        test_file = TEST_FOLDER / Path("modular_dialog_connector/connectionFields.xml")
```

Comment:
I think part of the magic of `Path` objects is that you can just add a string to the end and they'll make it part of the path.

```suggestion
        test_file = TEST_FOLDER / "modular_dialog_connector/connectionFields.xml"
```

can do the same below.